src/main/java/com/google/cloud/spanner/pgadapter/statements/IntermediateStatement.java

```diff
@@ -161,6 +161,10 @@ public Integer getUpdateCount() {
     return this.updateCount;
   }
 
+  public void addUpdateCount(int count) {
+    this.updateCount += count;
```

Comment:
is the updateCount ever used for anything? It seems like we're doing a lot of work to manage it but I  don't see how it's used

Reply:
This value is returned to the user at the end showing how many rows were updated.